autoload/vsnip/source.vim

```diff
@@ -9,10 +9,11 @@ endfunction
 "
 " vsnip#source#find.
 "
-function! vsnip#source#find(filetype) abort
+function! vsnip#source#find(bufnr) abort
+  let filetype = getbufvar(a:bufnr, '&filetype', '')
   let l:sources = []
-  let l:sources += vsnip#source#user_snippet#find(a:filetype)
-  let l:sources += vsnip#source#vscode#find(a:filetype)
+  let l:sources += vsnip#source#user_snippet#find(a:bufnr, filetype)
```

Comment:
Hi, I want to keep to the same for the `vscode` and `user_snippet` API.

I think your change is correct.
we should use `bufnr` instead of `filetype` for flexibility.

Sorry for a bit troublesome but Could you change as following?

1. Use bufnr instead of filetype in vscode source.
2. Remove `filetype` argument in user_snippet source.

Reply:
Sure, that makes a lot of sense.  Done